<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>728</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#label_title_login,QLabel#label_title_register{
	font: 25pt &quot;华文新魏&quot;;
}

QLabel{
	font: 14pt &quot;楷体&quot;;
}
QLineEdit{
       font: 10pt &quot;楷体&quot;;
	
 }

QPushButton{
	font: 12pt &quot;楷体&quot;;
      width:98px;
	height:40px;
	background-color: #fc5731;
}
QWidget#page_login{
    
	background-image: url(jka/jka.png);
}
QWidget#page_register{
	
	border-image: url(jka/01.png);
}



</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_register">
    <widget class="QPushButton" name="pushButton_register_submit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>530</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_register_return">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>530</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>150</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
</string>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_title_register">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>25</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
</string>
     </property>
     <property name="text">
      <string>注册界面</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pwd">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>210</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
</string>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
</string>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pwd2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>270</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>390</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
</string>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_phone">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>450</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);
</string>
     </property>
     <property name="text">
      <string>19572974232</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>450</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
</string>
     </property>
     <property name="text">
      <string>联系方式</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_num">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>330</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);
</string>
     </property>
     <property name="text">
      <string>201542340106</string>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>330</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
</string>
     </property>
     <property name="text">
      <string>学号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>390</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);
</string>
     </property>
     <property name="text">
      <string>吉凯铵</string>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_login">
    <widget class="QPushButton" name="pushButton_login_submit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>450</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_register">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>450</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);
</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_login_id">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>300</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>310</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);</string>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_title_login">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>200</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>25</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);

</string>
     </property>
     <property name="text">
      <string>登录界面</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_login_pwd">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>380</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>390</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);</string>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_pwd</tabstop>
  <tabstop>lineEdit_pwd2</tabstop>
  <tabstop>lineEdit_num</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_phone</tabstop>
  <tabstop>pushButton_register_submit</tabstop>
  <tabstop>pushButton_register_return</tabstop>
  <tabstop>pushButton_login_submit</tabstop>
  <tabstop>pushButton_register</tabstop>
  <tabstop>lineEdit_login_id</tabstop>
  <tabstop>lineEdit_login_pwd</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
